pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -22,6 +22,7 @@ import ElementCreationNew from "../common/elementCreationNew";
 import APIService from "../../apiService/apiService";
 import APIContext from "../../apiService/apiContext";
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
+import {TextareaGroup} from "../common/textareaGroup";
```

Comment:
The ticket is regarding **Endpoint** forms and not this form. So I think this implementation is not in the right spot.

I understand the reasoning behind the change, **please check with @rubenvdlinde if this change is wanted as we did not refine it.**